### PR TITLE
Add configuration of AutoLogin + Set AutoLogin duration as string + Update dev dependencies

### PR DESCRIPTION
**Pull request by vjik**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | -

### CODE REVIEW

File: config/params.php [hunk 1]
```diff
@@ -5,5 +5,8 @@
 return [
     'yiisoft/user' => [
         'authUrl' => '/login',
+        'autoLogin' => [
+            'duration' => new DateInterval('P5D'), // 5 days
```
Comment (roxblnfk) [inline config/params.php:9]: Can you define this value as scalar?

Comment (vjik) [reply config/params.php]: Done.

**Comment by samdark** [comment]
Related to https://github.com/yiisoft/user/pull/18. That's duplicate functionality, right?

**Comment by vjik** [comment]
> Related to #18. That's duplicate functionality, right?

No. This another...

File: src/AutoLogin.php [hunk 2]
```diff
@@ -19,9 +19,9 @@ final class AutoLogin
     private string $cookieName = 'autoLogin';
     private DateInterval $duration;
 
-    public function __construct(DateInterval $duration)
+    public function __construct(string $duration)
```
Comment (samdark) [inline src/AutoLogin.php:22]: What's the rationale behind this change?

**Review by samdark — APPROVED** [review]
